autoload/vista/finder.vim

```diff
@@ -153,8 +153,10 @@ function! vista#finder#PrepareOpts(source, prompt) abort
           \ }
 
   if exists('g:vista_fzf_preview')
+    let object_name_index = g:vista#renderer#enable_icon ? '2' : '1'
+    let extract_line_number = ':$(echo {' . object_name_index . "} | grep -o '[^:]*$')"
```

Comment:
Are you sure this also work on windows?

Reply:
Since I don’t own a windows machine it’s hard to tell, but since fzf.vim passes this command to [preview.sh](https://github.com/junegunn/fzf.vim/blob/master/bin/preview.sh) - a bash file, that to me confirms that we are in bash and have grep.

Reply:
I do not have a windows machine either, and I'm not using fzf in vim, I would leave this PR until someone can confirm this.